tools/docker-compose/README.md

```diff
@@ -505,32 +505,45 @@ Run a HashiVault container alongside of AWX.
 VAULT=true make docker-compose
 ```
 
-Go to `http://localhost:1234` sign in with method "Token".
+You can find the initialization data at `tools/docker-compose/_sources/secrets/vault_init.yml`,
+This includes the unseal keys and a root token.
 
-You can find the generated token at `tools/docker-compose/_sources/secrets/vault_password.yml`,
-this is a root token, and it should not need a corresponding username.
-Note that the token will be different on each restart, as it is re-generated by the playbook,
-and the container does not use a persistent volume.
+You will need to unseal the HashiVault each time the container is started.
+The easiest way to do that is to run:
+```bash
+ansible-playbook tools/docker-compose/ansible/unseal_vault.yml
```

Comment:
It looks like unsealing is done in the initialize playbook now? I'm not sure if this is necessary then.

Reply:
We have to do it in the initialize playbook so that we can create the default engine and secret.